teuthology/packaging.py

```diff
@@ -486,7 +487,7 @@ def _init_from_config(self):
         """
         Initializes the class from a teuthology job config
         """
-        self.arch = self.job_config.get('arch', 'x86_64')
+        self.arch = util.get_arch(self.job_config.get("machine_type"))
```

Comment:
This looks not good for me, because the util.get_arch makes a request to the lock server, which weird for constructor method from `job_config`. We should not be bound on lock server here at all.